lib/constants.js

```diff
@@ -67,4 +67,5 @@ export const PROGRESS_EXPLAINER_PATH = ENV.PROGRESS_EXPLAINER_PATH
 export const ONBOARDING_PACKAGES = ['ABO', 'BENEFACTOR', 'MONTHLY_ABO']
 
 export const TRIAL_CAMPAIGN = ENV.TRIAL_CAMPAIGN
+export const TRIAL_PAYNOTE_CAMPAIGN = ENV.TRIAL_PAYNOTE_CAMPAIGN
```

Comment:
wouldn't it be more consistent to use `TRIAL_CAMPAIGNS`?

Reply:
Yes-ish. Adopting.